roles/fio-bench/tasks/main.yml

```diff
@@ -25,21 +75,59 @@
           metadata:
             labels:
               app: fio-benchmark
-          spec:
-            containers:
-            - name: benchmark-server
-              image: "quay.io/benchmark-operator/fio:latest"
-              command: ["/bin/sh", "-c"]
-              args:
-                - "cat /tmp/fio/fiojob;
-                   fio /tmp/fio/fiojob"
-              volumeMounts:
-              - name: config-volume
-                mountPath: "/tmp/fio"
-            volumes:
-            - name: config-volume
-              configMap:
-                name: fio-test
-            restartPolicy: Never
-  when: fio.clients > 0
+      spec:
+        containers:
+        - name: bench-server
+          image: "quay.io/jtaleric/fio:testing"
+          command: ["/bin/sh","-c"]
+          args:
+           - "cat /tmp/fio/fiojob;
+              fio /tmp/fio/fiojob"
+          volumeMounts:
+          - name: config-volume
+            mountPath: "/tmp/fio"
+          - name: ceph-volume
+            mountPath: "/mnt/pvc"
+        volumes:
+        - name: config-volume
+          configMap:
+            name: fio-test
+        - name: ceph-volume
+          persistentVolumeClaim:
+            claimName: claim{{item}}
+        restartPolicy: Never
+  when: fio.clients > 0 and fio.storageclass is defined
+  with_sequence: start=1 count={{fio.clients}}
+
+- name: Start Server(s) - no PVC
+  k8s:
+    definition:
+      kind: Pod
```

Comment:
`kind: Job` still needs to be a job as it has a start time and an end time, and thus apiVersion also needs to be restored to `apiVersion: batch/v1`